### ISSUE TITLE
Communication to staff/observers

### ISSUE THREAD

**Issue by bast** [coderefinery/2021-05-10-workshop#40]
Subject: CodeRefinery: Connection details and practical info

Dear {first_name},

This is some of the last practical information for CodeRefinery starting next week (Monday, 8:50 CEST, 9:50 EEST, see time zone converter on workshop page):
https://coderefinery.github.io/2021-05-10-workshop/

Zoom connection details:
- link: (link)
- room ID: (id)
- passcode: (passcode)

Once you connect via Zoom, please rename your Zoom name to make exercise room management easier for us:
- If you are staff, your Zoom name is "(CR) {first_name} {last_name}"
- If you are observer, your Zoom name is "(OB) {first_name} {last_name}"
- If you can't figure out how to, we can help once you join.

Collaborative document for questions and notes (you will learn more about it on Monday):
(hackmd-link)

This workshop will be streamed (https://www.twitch.tv/coderefinery) and recorded. We have it set up so that you should not appear in the stream, however, to remove any chance of this happening you should leave your video off in the main room. However, any audio in the main room will be streamed to the world and recorded. If you prefer for this to not happen, the HackMD link (above) can be used to ask questions and interact. The breakout rooms are never recorded or streamed (and we encourage video and audio on there to be interactive). To learn about the streams, please see:
https://coderefinery.github.io/2021-05-10-workshop/join/

Our Code of Conduct: https://coderefinery.org/about/code-of-conduct/

If you don't wish to get emails about this workshop anymore, withdraw your registration using the modification link: {link}

Very much looking forward,
  Radovan Bast on behalf of CodeRefinery

**Comment by samumantha** [comment]
Subject: CodeRefinery: Day 2 follow up - instructors and expert helpers

Dear instructors and expert helpers,

Thank you for a great second workshop day! :)

fyi below day 2 follow up email that went out to all participants in the afternoon.

And a few notes:

First of all, thank you very much for all the updates in the coordination HackMD! This helps everyone to prepare for the next workshop day(s).

Instructors of day 3, please check the schedule published on the Workshop HackMD and adjust if needed: [workshop hackmd link]

All upcoming instructors:

* When sharing screen, especially the terminal, try to not fill the full shared window all the way to the bottom with the terminal , so that the zoom controls do not cover the important areas (where you are typing) for viewers.

* When stopping the screenshare, please announce that you will stop and wait a moment before you stop so that Richard can adjust settings for the stream.

Great work everyone!

See you tomorrow!

- Samantha

-------

Dear all,

Thank you for participating the first two days of the CodeRefinery workshop! We were happy to see and answer a lot of questions in the HackMD and see you discussing in the breakoutrooms!

If you for any reason missed the session today; you are more than welcome to join from Day 3! Today we continued with git (starting https://coderefinery.github.io/git-intro/06-branches/), and learned about branching, merging, and history inspection. All Q&A from todays HackMD can be found here: https://coderefinery.github.io/2021-05-10-workshop/questions/day2/.

Again, please do not get discouraged if not everything was 100% clear to you. As with everything, learning new things takes time and repetition. You may rewatch  the workshop videos on twitch (https://www.twitch.tv/coderefinery) for the next two weeks. And we will use some of the commands again tomorrow, to get used to them.

Tomorrow you'll continue to see the power of a version control system, as we will proceed to truly collaborative work (https://coderefinery.github.io/git-collaborative/).

For tomorrow, it will be important that everyone has the ssh keys for github set up  and working. Please follow the instructions to also test if things are set up correctly: https://coderefinery.github.io/installation/ssh/.

Planned schedule day 3:
- 08:50 Soft start
- 09:00-09:15 Welcome and summary
- 09:15-09:45 Concepts around collaboration
- 09:45-10:05 Centralized workflow
- 10:05-10:15 **Break**
- 10:15-10:55 Centralized workflow exercise exercise  & Q&A
- 10:55-11:15 Distributed version control and forking workflow
- 11:15-11:25 **Break**
- 11:25-12:10 Practice collaborative forking workflow exercise & Q&A
- 12:10-12:25 How to contribute changes to somebody else's project
- 12:25-12:30 Summary, Q&A, what's next week  
- End 12:30

All other than the mentioned exercises here will be optional, if you have time in the breakoutrooms.

Thank you all again for the constructive feedback. We apologize for the hassle with turning on and off the zoom videos. Background: In order to ensure that no participant will appear on stream/video, zoom video of participants needed to be disabled in the main room. If participants disabled video by themselves, everything was fine. Only if the host had to disable video for participants, then it was not possible to turn it back on by participants themselves and a host had to come to the breakoutroom and click a button. **Therefore we kindly ask you to always turn your video back off when joining the main room.**

[links to hackmd, twitch, zoom]

See you tomorrow!

Best regards,
your CodeRefinery team

PS: If you don't want to get these emails anymore, you can unsubscribe from this link: {link}

**Comment by samumantha** [comment]
Subject: CodeRefinery: Day 1 follow up - instructors 

Dear expert helpers and instructors,

thank you for a great first day of the workshop!

fyi below the email that went out to every participant in the afternoon.

We will try to send out the schedule for the next day every afternoon before next workshop day, so (@instructors) please talk to other instructors of the same day and update the break/exercise schedule for your lesson in the coordination hackmd: [link to coordination hackmd] . If you want any specific instructions shared with participants the day before, use #workshops/2021-may 20 communication stream topic in zulip chat to let us know.

Thanks and see you tomorrow!

Samantha

-----

Dear all,

Thank you for participating the first day of the CodeRefinery workshop! It was great to see a lot of interesting questions (that hopefully got satisfactorily answered) and the interaction happening in the breakout rooms. Special thanks to all the exercise leaders! Also thank you for the feedback, we will try to implement it for the upcoming days, let us know how we did in the next feedback round :)

Please do not get discouraged if not everything was 100% clear to you. As with everything, learning new things takes time and repetition. You may re-watch parts of the course on twitch (https://www.twitch.tv/videos/1017579131) for the next two weeks.

If you for any reason missed the session today; you are more than welcome to join from Day 2! Today we learned from "Motivation" to "Undoing things" in Introduction to version control with Git lesson. Please refer to the lesson material (https://coderefinery.github.io/git-intro/) and the archived HackMD (https://coderefinery.github.io/2021-05-10-workshop/questions/day1/) for your review .

Tomorrow, we continue with git (starting https://coderefinery.github.io/git-intro/06-branches/), and do branching, merging, and history inspection. You'll start to see the power of a version control system, as we prepare for truly collaborative work on day 3.

Planned schedule day 2 (with Q&A buffer at end of each episode, all times in CEST):

    08:50 Soft start
    09:00-09:15 Welcome and Summary of day 1
    09:15-09:55 Branching and merging, one Exercise session at ~09:25
    09:55-10:05 Break
    10:05-10:40 Conflict resolution, one Exercise session at ~10:15
    10:40-11:05 Sharing repositories online
    11:05-11:15 Break
    11:15-12:00 Inspecting history, one Exercise session at ~ 11.30
    12:00-12:10 Break
    12:10-12:30 Summary, Q&A, next day preparations
    12:30 End 

Upon request we will leave the breakoutrooms open during breaks for you to chat with your colleagues or further discuss the exercises. However, we strongly advice everyone to also take a screen break during that time!

We are very sorry for any technical issues that made it hard to follow for some of you (screenshare freeze, breakoutroom trouble, HackMD freeze etc). We will try to minimize these as much as possible from our side. In case you missed parts of the course, please refer to the recordings and use the HackMD to ask for clarifications. We want you to get the most out of this workshop. No question is too basic/stupid!

For those of you that are overwhelmed with the flood of information and number of tabs/windows:
We suggest to follow the instructors screenshare (half screen) and following type along in own terminal. HackMD and lesson material can be in the background, to refer to if there is questions/when exercises are announced. HackMD, lesson material and recordings of all sessions will also be available after the workshop.

Please be aware that we may need to rearrange breakout room teams (due to exercise leader/participant unavailability) to make sure we always have at least one exercise leader in each room and have a minimum of 3 people per room to encourage discussions. Sorry for any inconvenice that may arise from this.

See you tomorrow (Zoom and HackMD link stays the same!)
Best regards,
your CodeRefinery team

PS: If you don't want to get these emails anymore, you can unsubscribe from this link:{link}
